File: scala/src/test/scala/cucumber/runtime/scala/test/RunCukesTest.scala
package cucumber.runtime.scala.test

import _root_.org.junit.runner.RunWith
import _root_.cucumber.api.junit.Cucumber
import gherkin.formatter.model.Feature

@RunWith(classOf[Cucumber])
@Cucumber.Options(tags = Array("@SmokeTest"), strict=true)
class RunCukesTest

File: hk2/src/test/java/cucumber/runtime/java/hk2/RunCukesTest.java
package cucumber.runtime.java.hk2;

import cucumber.api.CucumberOptions;
import cucumber.api.junit.Cucumber;
import org.junit.runner.RunWith;

@RunWith(Cucumber.class)
@CucumberOptions(glue = "cucumber.runtime.java.hk2.integrationTest")
public class RunCukesTest {
}
